scripts/.env-example

```diff
@@ -59,6 +59,10 @@ BPA_CREDDEF_REVOCATION_REGISTRY_SIZE=3000
 
 ACAPY_GENESIS_URL=https://indy-test.bosch-digital.de/genesis
 
+# To support revocation set the 2 URLs of an existing tails server
+#ACAPY_TAILS_BASE_URL=--tails-server-base-url <base url>
+#ACAPY_TAILS_UPLOAD_URL=--tails-server-upload-url <upload url>
```

Comment:
To keep it simple I would not expose the upload url here, as we have enough properties already :) If not set the upload url defaults to the base url. And if one wants to set it separately one can override it in the helm chart.